Mailbox.Tests/DataLoader.Tests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+#pragma warning disable CS8625
+#pragma warning disable CS8602
+#pragma warning disable CS8600
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void DataLoader_Constructor_NullStream()
+        {
+            DataLoader loader = new DataLoader(null);
+        }
+
+        [TestMethod]
+        public void DataLoader_Save_Success()
+        {
+            List<Mailbox> boxList = new List<Mailbox>();
+            MemoryStream source = new MemoryStream();
+            DataLoader loader = new DataLoader(source);
+
+            boxList.Add(new Mailbox(Size.Small, new ValueTuple<int, int>(1, 1), new Person("Some", "One")));
```

Comment:
Value tuple can be simplified.
```suggestion
            boxList.Add(new Mailbox(Size.Small, (1, 1), new Person("Some", "One")));
```